internal/conditions/error_to_message.go

```diff
@@ -0,0 +1,39 @@
+package conditions
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/telemetry-manager/internal/workloadstatus"
+)
+
+type ErrorToMessageConverter struct {
+}
+
+func (etc *ErrorToMessageConverter) Convert(err error) string {
```

Comment:
lets have some documentation on either the implementation or our interface about the purpose of this function